assembly/src/parsers/u32_ops.rs

```diff
@@ -88,29 +88,23 @@ pub fn parse_u32assert(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(),
 
 /// Translates u32assert assembly instruction to VM operations.
 ///
-/// Implemented by executing `U32SPLIT EQZ ASSERT` on each element in the word.
+/// Implemented by executing `PAD U32ASSERT2 DROP` on each element in the word.
```

Comment:
very minor nit: `PAD U32ASSERT2 DROP` is not executed on each element in the word - but on each pair of elements in the word.